src/gui/mzroll/peakdetectiondialog.cpp

```diff
@@ -183,6 +196,21 @@ PeakDetectionDialog::PeakDetectionDialog(MainWindow* parent) :
                 mainwindow->massCalcWidget->fragPpm,
                 SLOT(setValue(double)));
 
+        slider = new RangeSlider(Qt::Horizontal, RangeSlider::Option::DoubleHandles, this);
+        gridLayout_6->addWidget(slider);
+        connect(peakMl, &QGroupBox::toggled,
+                [this](const bool checked)
+                {
+                    if(checked){
+                        getLoginForPeakMl();
+                    }
+                    else{
+                        peakMlSet = false;
+                        mainwindow->mavenParameters->peakMl = false;
+                        modelTypes->setEnabled(false);
+                    }
```

Comment:
Let's add analytics here too? Mixpanel would be preferable over GA. Since it gives us a better idea of who was interested in PeakML. But as I mentioned before, the current label for this group-box does not really hint at some cool ML tech.

Reply:
Haven't added it till now, was facing an issue. It will be resolved asap.